charts/s3gw/values.yaml

```diff
@@ -45,7 +47,7 @@ ui:
   # 'serviceName' is the service name of the S3GW user interface.
   serviceName: "s3gw-ui"
   # 'publicDomain' is the public domain of the UI Service used by the Ingress.
-  publicDomain: "fe.127.0.0.1.omg.howdoi.website"
+  publicDomain: "example.com"
```

Comment:
I'd rather prefer the user to be forced to put an explicit value.
I think it is better, for the public domain variables, to require the user to make the choice.

We can implement the following logic inside a `chart-validations.yaml` file:

```
{{- if (empty .Values.ui.publicDomain) }}
{{- fail "ui.publicDomain cannot be empty" }}
{{- end }}
```

Reply:
Sure, sounds like a good idea.
Can you point me to the docs for the `chart-validation.yaml` file? I only found https://helm.sh/docs/topics/charts/#schema-files schema files, which don't seem to be the same thing.

Reply:
we did a very similar thing in Epinio, looks here: https://github.com/epinio/helm-charts/blob/main/chart/epinio/templates/chart-validations.yaml